model/annotations_schema.js

```diff
@@ -7,74 +8,109 @@ sql.setDialect('postgres')
-	{name: 'asa_u_id',       dataType: 'serial', primaryKey:true},
-	{name: 'user_id',       dataType: 'varchar(40)', notNull: true},
-    ]});
-// todo: maintain unique key constraint with the source?
-
-interactions = sql.define({
-    name: 'ppi_annos',
+      {name: 'user_id',       dataType: 'varchar(40)', notNull: true},
+      {name: 'interaction_id', dataType: 'integer'},
+      {name: 'identifier',	dataType: 'varchar(45)', notNull: true},
+      {name: 'id',       dataType: 'integer'}],
+})
+
+exports.interactions = interactions = sql.define({
+    name: 'annotations_interaction',
     columns: [
-	{name: 'source',	dataType: 'varchar(15)', notNull: true},
-	{name: 'target',	dataType: 'varchar(15)', notNull: true},
-	{name: 'database',	dataType: 'varchar(30)'},
-	{name: 'type',	 dataType: 'varchar(15)'},
-	{name: 'weight', dataType: 'float'},
-	{name: 'directed',	dataType: 'boolean'},
-	{name: 'tissue',	dataType: 'varchar(30)'},
-	{name: 'anno_type',	dataType: annoTypeName + " DEFAULT 'ppi'", notNull:true},
-	{name: 'anno_id', dataType: 'integer', primaryKey: true, references: {table: 'annos', column: 'u_id', onDelete: 'cascade'}}]
+	{name: 'source_id',	dataType: 'varchar(15)', notNull: true},
+	{name: 'target_id',	dataType: 'varchar(15)', notNull: true},
+	{name: 'input_source',	dataType: 'varchar(30)'},
+	{name: 'id', dataType: 'integer', primaryKey: true}]
 })
 
-votes = sql.define({
-    name: 'votes',
+exports.votes = votes = sql.define({
+    name: 'annotations_interactionvote',
     columns: [
-	{name: 'anno_id', dataType: 'integer', primaryKey: true, references: {table: 'annos', column: 'u_id', onDelete: 'cascade'}},
-	{name: 'anno_type',	dataType: annoTypeName, notNull:true, primaryKey: true},
-	{name: 'voter_id', dataType: 'varchar(40)', notNull: true, primaryKey: true},
-	// integrity check: only one vote at a time
-	{name: 'direction', dataType: 'smallint', notNull: true},
-	{name: 'comment', dataType: 'varchar(3000)'}]
+	{name: 'id', dataType: 'integer', primaryKey: true},
+	{name: 'reference_id',	dataType: 'integer', notNull:true},
+	{name: 'user_id', dataType: 'varchar(40)', notNull: true},
+	{name: 'is_positive', dataType: 'smallint', notNull: true}]
+})
+
+exports.users = users = sql.define({
+    name: 'auth_user',
+    columns: [
+	{name: 'id', dataType: 'serial', primaryKey: true},
+	{name: 'password',	dataType: 'varchar(128)', notNull: true},
+	{name: 'last_login', dataType: 'timestamp'},
+	{name: 'username', dataType: 'varchar(30)', notNull: true},
+	{name: 'first_name', dataType: 'varchar(30)', notNull: true},
+	{name: 'last_name', dataType: 'varchar(30)', notNull: true},
+	{name: 'email', dataType: 'varchar(254)', notNull: true},
+	{name: 'is_staff', dataType: 'boolean', notNull: true},
+	{name: 'is_superuser', dataType: 'boolean', notNull: true},
+	{name: 'is_active', dataType: 'boolean', notNull: true},
+	{name: 'date_joined', dataType: 'timesamp', notNull: true}
```

Comment:
timestamp contains both date and time; date just has the day.  for some reason that's what appeared in the django database so i followed it.